.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,298 @@
+eclipse.preferences.version=1

```

Comment:
This may be helpful http://stackoverflow.com/questions/1143796/remove-a-file-from-a-git-repository-without-deleting-it-from-the-local-filesyste

git rm --cached -r mydirectory